presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -137,17 +165,20 @@ public ThriftHiveMetastoreStats getStats()
         return stats;
     }
 
+    @VisibleForTesting
+    public List<ThriftHiveMetastoreStats> getAllStats()
```

Comment:
Why not to return it as a map? It is very confusing what does this method return. How about returing a map, and calling the method `Map<HostAndPort, ThriftHiveMetastoreStats> getMetastoreHostStats`